LICENSE.md

```diff
@@ -0,0 +1,13 @@
+Copyright 2023 OpenBioML
```

Comment:
Small nitpick, but I'd advocate in favor of making it more precise in noting that the copyright belongs to repo contributors, e.g. something like "Copyright 2023 the authors". OpenBioML works as well for me though.

Reply:
Thanks, I've updated it to say "the authors"!